nls/kappnav.properties

```diff
@@ -230,10 +230,10 @@ action.url.deployment.projdoc.text= View Project Doc
 action.url.deployment.projdoc.desc= View project documentation.
 action.url.deployment.projcode.text= View Project Code
 action.url.deployment.projcode.desc= View project source code.
-action.url.deployment.kabcolldoc.text= View Collection Doc
-action.url.deployment.kabcolldoc.desc= View Kabanero Collection Documentation.
-action.url.deployment.kabcollcode.text= View Collection Code
-action.url.deployment.kabcollcode.desc= View Kabanero collection source code.
+action.url.deployment.kabcolldoc.text= View Stack Doc
```

Comment:
We should rename the keys from having `coll` to `stack`.  It's a minor observation that doesn't impact code functionality

Reply:
Thanks, Kin.  I agree.  I wasn't entirely sure those keys weren't being used in code somewhere so I wanted to make minimal changes this close to the release.  I'll update the keys after the release.